Tests/SonarQube.Bootstrapper.Tests/BootstrapperSettingsTests.cs

```diff
@@ -70,15 +43,14 @@ public void BootSettings_DownloadDirFromEnvVars()
                 AssertExpectedDownloadDir(@"tfs build\.sonarqube\bin", settings);
             }
 
-            // 3. SQ variable takes precedence over the other variables
+            // 3. CWD has least precedence over env variables
             using (EnvironmentVariableScope scope = new EnvironmentVariableScope())
             {
-                scope.SetVariable(BootstrapperSettings.SQAnalysisRootPath, "sq build");
-                scope.SetVariable(BootstrapperSettings.BuildDirectory_Legacy, "legacy tf build");
-                scope.SetVariable(BootstrapperSettings.BuildDirectory_TFS2015, "tfs build");
+                scope.SetVariable(BootstrapperSettings.BuildDirectory_Legacy, null);
+                scope.SetVariable(BootstrapperSettings.BuildDirectory_TFS2015, null);
 
                 IBootstrapperSettings settings = new BootstrapperSettings(logger);
-                AssertExpectedDownloadDir(@"sq build\.sonarqube\bin", settings);
+                AssertExpectedDownloadDir(Path.Combine(Directory.GetCurrentDirectory(), @".sonarqube\bin"), settings);
```

Comment:
Sorry, Microsoft slang :)

I meant it's low priority

Sent from my Windows Phone
________________________________
From: Dinesh Bolkensteyn<mailto:notifications@github.com>
Sent: ‎11/‎06/‎2015 15:27
To: SonarSource/sonar-msbuild-runner<mailto:sonar-msbuild-runner@noreply.github.com>
Cc: Bogdan Gavril<mailto:bgavril@microsoft.com>
Subject: Re: [sonar-msbuild-runner] SONARMSBRU-43 Use the .sonarqube folder in CWD in the bootstrapper (#18)


In Tests/SonarQube.Bootstrapper.Tests/BootstrapperSettingsTests.cs<https://github.com/SonarSource/sonar-msbuild-runner/pull/18#discussion_r32225810>:

>
>                  IBootstrapperSettings settings = new BootstrapperSettings(logger);
> -                AssertExpectedDownloadDir(@"sq build\.sonarqube\bin", settings);
> +                AssertExpectedDownloadDir(Path.Combine(Directory.GetCurrentDirectory(), @".sonarqube\bin"), settings);


P3?

—
Reply to this email directly or view it on GitHub<https://github.com/SonarSource/sonar-msbuild-runner/pull/18/files#r32225810>.